### PR TITLE
Replace boilerplate RepoName with FaithlifeMath.

### PR DESCRIPTION
**Pull request by ChristianAlexander**

### CODE REVIEW

File: src/Faithlife.Math/Faithlife.Math.csproj [hunk 1]
```diff
@@ -13,10 +13,10 @@
...
     <PackageReleaseNotes>https://github.com/Faithlife/FaithlifeMath/VersionHistory.md</PackageReleaseNotes>
     <RepositoryUrl>https://github.com/Faithlife/FaithlifeMath.git</RepositoryUrl>
-    <PackageLicenseUrl>https://raw.githubusercontent.com/Faithlife/RepoName/master/LICENSE</PackageLicenseUrl>
-    <PackageProjectUrl>https://github.com/Faithlife/RepoName</PackageProjectUrl>
-    <PackageReleaseNotes>https://github.com/Faithlife/RepoName/blob/master/VersionHistory.md#released</PackageReleaseNotes>
-    <RepositoryUrl>https://github.com/Faithlife/RepoName.git</RepositoryUrl>
+    <PackageLicenseUrl>https://raw.githubusercontent.com/Faithlife/FaithlifeMath/master/LICENSE</PackageLicenseUrl>
+    <PackageProjectUrl>https://github.com/Faithlife/FaithlifeMath</PackageProjectUrl>
+    <PackageReleaseNotes>https://github.com/Faithlife/FaithlifeMath/blob/master/VersionHistory.md#released</PackageReleaseNotes>
+    <RepositoryUrl>https://github.com/Faithlife/FaithlifeMath.git</RepositoryUrl>
```
Comment (bgrainger) [inline src/Faithlife.Math/Faithlife.Math.csproj]: These four lines need to be merged with the four above, keeping the most recent version of each. This was a bad merge in 60741806af274aabc3791d04c800d9c36e6d60a3.

**Review by bgrainger — CHANGES_REQUESTED** [review]
Need to fix the duplicate elements in the CSPROJ (which is strangely not a compiler error).

File: src/Faithlife.Math/Faithlife.Math.csproj (hunk 1 above)
Comment (ChristianAlexander) [reply src/Faithlife.Math/Faithlife.Math.csproj]: Oh, I hadn't seen that. That really *is* strange that it's not a compiler error.

File: src/Faithlife.Math/Faithlife.Math.csproj [hunk 2]
```diff
@@ -13,10 +13,6 @@
     <PackageProjectUrl>https://github.com/Faithlife/FaithlifeMath</PackageProjectUrl>
     <PackageReleaseNotes>https://github.com/Faithlife/FaithlifeMath/VersionHistory.md</PackageReleaseNotes>
     <RepositoryUrl>https://github.com/Faithlife/FaithlifeMath.git</RepositoryUrl>
-    <PackageLicenseUrl>https://raw.githubusercontent.com/Faithlife/RepoName/master/LICENSE</PackageLicenseUrl>
-    <PackageProjectUrl>https://github.com/Faithlife/RepoName</PackageProjectUrl>
-    <PackageReleaseNotes>https://github.com/Faithlife/RepoName/blob/master/VersionHistory.md#released</PackageReleaseNotes>
```
Comment (bgrainger) [inline src/Faithlife.Math/Faithlife.Math.csproj]: We need the fix in this line from https://github.com/Faithlife/CSharpTemplate/commit/5d141352f477c59bbc4a8eb5f34bd822d8d3dd6d

It has to be merged with the right repo name on line 14.

**Review by bgrainger — CHANGES_REQUESTED** [review]
The bad merge kept both sets of changes; we need the results of performing the merge correctly.

**Review by bgrainger — APPROVED** [review]

**Pull request merged**